lib/fastlane/plugin/wpmreleasetoolkit/helper/configure_helper.rb

```diff
@@ -225,6 +225,10 @@ def self.add_file(params)
 
         UI.user_error!('You must pass a `destination` to `add_file`') unless params[:destination]
 
+        unless Fastlane::Helper::GitHelper.is_ignored?(path: params[:destination])
+          UI.user_error! "Attempted to add a file to a location which is not ignored under Git (#{params[:destination]}). Please edit your `.gitignore` manually to fix this."
```

Comment:
```suggestion
          UI.user_error! "Attempted to add a file to a location which is not ignored under Git (#{params[:destination]}). Please either edit your `.configure` file to use an already-ignored destination, or add that destination `.gitignore` manually to fix this."
```
I know it's a bit lengthy, but I prefer more hints/documentation than having people (including future us) losing time wondering what's the expected fix 🙂 

PS: Feel free to reword, or introduce a `\n` somewhere in the message, and/or use another ruby string syntax (e.g. heredoc) to format this in a way that makes rubocop happy if needed.